docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -691,6 +693,78 @@ System.out.println(description.writeQueries().size() + " queries write to this s
 System.out.println(description.readQueries().size() + " queries read from this stream/table.");
 ``` 
 
+Get metadata about the ksqlDB cluster
+------------------------------------------------------------------
+
+Starting with ksqlDB 0.16.0, the `serverInfo()` method enables client apps to fetch metadata about
```

Comment:
Ideally this piece would be in a separate PR so we can get the change live for 0.16 ahead of the connector docs, but it's not a huge deal; your call on whether you think it's worth the effort.

Reply:
Joel and I had a chat with MD and Apurva, and we're trying to get me out of the Mechanical Turk business. :)

Reply:
Since the rest of the docs will be coming out for 0.18, I'll move this to another PR.